treeherder/model/management/commands/cycle_data.py

```diff
@@ -82,40 +81,27 @@ class PerfherderCycler(DataCycler):
     def __init__(self, days, chunk_size, sleep_time, is_debug=None,
                  logger=None, **kwargs):
         super().__init__(days, chunk_size, sleep_time, is_debug, logger)
-        self.unsheriffed_repos_expire_days = datetime.timedelta(days=kwargs['days_for_nonmain'])
         self.started_at = None
 
-    @property
-    def should_quit(self):
-        now = datetime.datetime.now()
-        elapsed_runtime = now - self.started_at
-
-        return self.max_runtime < elapsed_runtime
-
     def cycle(self):
         self.started_at = datetime.datetime.now()
+        max_chunk_size = self.chunk_size
```

Comment:
Could you make these names the same for consistency?

Reply:
Sure thing.